### PR TITLE
Fix createElementFromAttrs bug

### PR DESCRIPTION
**Pull request by wxiaoguang**
The "false" value was not handled correctly, it would cause bugs in the future (fortunately, this behavior is not used in code yet).

### CODE REVIEW

File: web_src/js/utils/dom.ts [hunk 1]
```diff
@@ -297,7 +297,7 @@ export function createElementFromAttrs(tagName, attrs) {
   const el = document.createElement(tagName);
   for (const [key, value] of Object.entries(attrs)) {
     if (value === undefined || value === null) continue;
-    if (value === true) {
+    if (value === true || value === false) {
```
Comment (silverwind) [inline web_src/js/utils/dom.ts]:
```suggestion
    if (typeof value === "boolean") {
```

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: image

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: image

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: image

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: image

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: image

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: image

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: ps: Using `===` for `data.find(x => typeof x === "boolean")` doesn't make difference.

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]:
```suggestion
    if (typeof value === 'boolean') {
```

Comment (wxiaoguang) [reply web_src/js/utils/dom.ts]: By testing with more cases and browsers, it seems that there is no always-winner. I guess it is related to that the browser's JIT generates different machine code for different logic.

But I still dislike the `typeof` here, it introduces a string literal `'string'` which is unnecessary. `=== true ||  === false` is clear enough.

If you'd really like to change it, the string needs to be quoted by a single quote (done in 82510da3ef85232fc88241af3f0e5c53ad986803)

**Review by delvh — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**